### DISCUSSION TITLE
Would it be possible to resize the image view window?

### DISCUSSION THREAD

**Discussion by GalaxyTimeMachine** [hlky/stable-diffusion-webui#569] [Feature suggestions]
The image size window is sometimes too small to make out the detail, and I have to keep opening files from the output directory. Would it be possible to add a drag to resize type window that the image will expand into?

**Comment by Jonseed** [comment]
That would be good, or an option to show the image in its native resolution (100% zoom), so a 512x512 image shows at that resolution.
